docs/_user_guide/data_transfer.md

```diff
@@ -1,339 +1,257 @@
 
----
-<a href="/gettingStarted/6_filesystem.html"><span class="glyphicon glyphicon-menu-left fa-lg" style="float: left;"/></a> <a href="/gettingStarted/8_jupyterhub.html"><span class="glyphicon glyphicon-menu-right fa-lg" style="float: right;"/></a>
+## General Guidelines
+| **File Size/Transfer Scenario**            | **Recommended Methods**                                                                 | **Explanation**                                                                                                                                                                |
+|--------------------------------------------|----------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Files less than 1 GB**                   | Upload via JupyterHub or use `scp`.                                                    | For small files, simple methods like uploading directly through JupyterHub or using `scp` (Secure Copy Protocol) are quick, straightforward, and efficient.                   |
+| **Files between 1 GB and 10 GB**           | Use `scp` over the Data Transfer Node (`yen-transfer`).                                 | Transferring directly to the Data Transfer Node ensures faster speeds and reduces load on the interactive nodes. `scp` remains effective for these moderately sized files.     |
+| **Files between 10 GB and 50 GB**          | Use `rsync` over the Data Transfer Node or `scp` with the `-C` (compression) option.    | `rsync` is more efficient for larger datasets because it can resume interrupted transfers and only copies changed parts of files. Using `scp -C` enables compression.          |
+| **Files between 50 GB and 1 TB**           | Use Globus or `rsync` over the Data Transfer Node.                                      | Globus is designed for transferring large datasets reliably and efficiently. It handles network interruptions gracefully and optimizes transfer settings for large files.       |
+| **Files larger than 1 TB**                 | Use Globus.                                                                            | For very large datasets, Globus provides high-performance, secure transfers with features like parallelism and checkpointing, which are essential for handling transfers of this scale. |
+| **Cloud Transfers**                        | Use RClone with Google Drive for archiving                                      | RClone efficiently syncs data to cloud storage providers, making it suitable for archiving or cloud backups|                                                                                                                                                                          |
```

Comment:
Use RClone with Google Drive for archiving  --> Use RClone with a cloud "remote"